src/components/footer.tsx

```diff
@@ -0,0 +1,29 @@
+import { FaGithub } from 'react-icons/fa';
+import { ExternalLink } from 'src/components';
+
+export const Footer: React.FC = () => (
+  <footer
+    className="absolute text-white bottom-0 p-3 pb-6 lg:pb-2 text-sm lg:text-base
+    flex flex-col items-center justify-center w-full h-16 sm:h-20 text-center  pointer-events-none"
+  >
+    <div className="flex items-center leading-relaxed py-1 lg:py-0">
+      ©2020
+      <ExternalLink className="underline hover:opacity-50 mx-1 pointer-events-auto" href="https://twitter.com/placeholder">
+        placeholder
+      </ExternalLink>
+      <ExternalLink className="underline hover:opacity-50 mr-1 pointer-events-auto" href="https://github.com/placeholder">
```

Comment:
ごめんここ適当に書いたの忘れてた、だれを書くといいだろう
ワイの名前を適当にかいてもいいけど

```suggestion
      <ExternalLink className="underline hover:opacity-50 mx-1 pointer-events-auto" href="https://twitter.com/hpp_ricecake">
        placeholder
      </ExternalLink>
      <ExternalLink className="underline hover:opacity-50 mr-1 pointer-events-auto" href="https://github.com/nama-tamago">
```

Reply:
https://twitter.com/p1ass にしておこう

Reply:
あ、確かにそれがいいね、変えておきます